File: swarm-keydb-js/examples/config-management.mjs
import { SwarmKeyDb } from '../index.js';

const db = new SwarmKeyDb({ host: '127.0.0.1', port: 6379 });
await db.connect();
await db.batchPut({ 'config:theme': 'dark', 'config:region': 'eu-west-1' });
console.log(await db.batchGet(['config:theme', 'config:region']));
await db.disconnect();

File: swarm-keydb-py/examples/config_management.py
from swarm_keydb import SwarmKeyDb

client = SwarmKeyDb(host="127.0.0.1", port=6379)
client.batch_put({"config:theme": "dark", "config:region": "eu-west-1"})
print(client.batch_get(["config:theme", "config:region"]))
